core/src/main/kotlin/com/strumenta/kolasu/model/Printing.kt

```diff
@@ -23,6 +23,7 @@ fun Node.relevantMemberProperties(withPosition: Boolean = false, withNodeType: B
                 it.name != "origin" &&
                 it.name != "destination" &&
                 it.name != "sourceText" &&
+                it.name != "source" &&
```

Comment:
This is a method that we should probably revise at some point to use the Internal annotation, but I am adding this just as a reminder, not to have this fixed as part of this PR